### ISSUE TITLE
Troubleshoot Function App not working after initial build

### ISSUE THREAD

**Issue by adamrushuk** [adamrushuk/devops-lab#113]
The Function App should start straight away, but requires manually restarting app service before it starts working.
I thought this was a known issue (terraform-provider-azurerm issue 8546) but it's not quite the same behaviour.

- [x] Add a script to restart App Service?
- [x] check export template JSON
- [x] Recreate function app using latest vscode extension
- [ ] Raise a separate issue with specific details, like: "PowerShell Core Version" value is blank within Configuration > General settings tab.

**Comment by adamrushuk** [comment]
Check passing powershellversion site config: https://github.com/neumanndaniel/armtemplates/blob/aa14255f3a2be592ac60dc8ff541fda7cb10c649/aks-snapshot-backup/aks-snapshot-backup-infrastructure.json#L97

**Comment by adamrushuk** [comment]
Waiting on response from known issue: https://github.com/terraform-providers/terraform-provider-azurerm/issues/8867

**Comment by adamrushuk** [comment]
Just noticed this started working, despite missing `siteConfig.PowerShellVersion`. It just seems to takes 10-15 mins to kick in.

**Comment by adamrushuk** [comment]
This issue is intermittent, so might be worth using PowerShell 6.2 until 7.0 becomes the default
